editattendees.php

```diff
@@ -105,20 +106,30 @@
                 $erruser = $DB->get_record('user', array('id' => $adduser), "id, {$usernamefields}");
                 $errors[] = get_string('error:addalreadysignedupattendee', 'facetoface', fullname($erruser));
             } else {
-                if (!facetoface_session_has_capacity($session, $context)) {
-                    $errors[] = get_string('full', 'facetoface');
-                    break; // No point in trying to add other people.
-                }
-                // Check if we are waitlisting or booking.
-                if ($session->datetimeknown) {
-                    $status = MDL_F2F_STATUS_BOOKED;
-                } else {
-                    $status = MDL_F2F_STATUS_WAITLISTED;
-                }
-                if (!facetoface_user_signup($session, $facetoface, $course, '', MDL_F2F_BOTH,
-                $status, $adduser, !$suppressemail)) {
-                    $erruser = $DB->get_record('user', array('id' => $adduser), "id, {$usernamefields}");
-                    $errors[] = get_string('error:addattendee', 'facetoface', fullname($erruser));
+                $now = time();
+                $addtofuturesessions = $addtoallsessions && $facetoface->signuptype == MOD_FACETOFACE_SIGNUP_MULTIPLE;
+                $futuresessions = array_filter(
```

Comment:
should we use facetoface_get_future_sessions here?

Reply:
Oops, yes we should. Fixed that.